src/Controls/samples/Controls.Sample/wwwroot/index.html

```diff
@@ -6,6 +6,7 @@
     <title>Blazor Android app</title>
     <base href="/" />
     <link href="css/app.css" rel="stylesheet" />
+    <link href="_content/MauiRazorClassLibrarySample/MauiRazorClassLibrarySample.bundle.scp.css" rel="stylesheet" />
```

Comment:
This is wrong I believe. You should reference `Controls.Sample.styles.css`, the `.bundle.scp.css` files only contains the bundle for the scoped css files, it doesn't have import references to other libraries. This works right now because you don't have transitive RCLs in the app.

Reply:
For others watching: I met with @javiercn and @Redth today and we have a different solution that should fix this.